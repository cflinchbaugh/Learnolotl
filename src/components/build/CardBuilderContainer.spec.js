import React from 'react';
import {shallow} from 'enzyme';
import {CardBuilderContainer} from './CardBuilderContainer';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('CardBuilderContainer', () => {
    it('renders and matches snapshot', () => {
        const renderer = new ShallowRenderer();

        const tree = renderer.render(
            <CardBuilderContainer />
        );

        expect(tree).toMatchSnapshot();
    });

});  