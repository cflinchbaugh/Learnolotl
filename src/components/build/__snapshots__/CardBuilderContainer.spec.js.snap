// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardBuilderContainer renders and matches snapshot 1`] = `
<CardBuilder
  exportButtonData={
    Object {
      "label": "Download",
      "onClickFunction": [Function],
      "type": "next",
    }
  }
  formElementFactoryData={
    Object {
      "formElementData": undefined,
      "formElements": Array [],
      "handleInputChange": [Function],
    }
  }
  handleSubmitForm={[Function]}
  nextCardButtonData={
    Object {
      "label": "Next Card",
      "type": "submit",
    }
  }
/>
`;
